asistente_ladm_col/logic/quality_rules/qr_validate_legal_party.py

```diff
@@ -0,0 +1,190 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-06-15
+        git sha         : :%H$
+        copyright       : (C) 2022 by Leonardo Cardona (SwissTierras Colombia)
+        email           : contacto@ceicol.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from qgis.core import QgsVectorLayer
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4008,
+                                                           QRE_IGACR4008E01,
+                                                           QRE_IGACR4008E02,
+                                                           QRE_IGACR4008E03,
+                                                           QRE_IGACR4008E04,
+                                                           QRE_IGACR4008E05,
+                                                           QRE_IGACR4008E06,
+                                                           QRE_IGACR4008E07)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.lib.geometry import GeometryUtils
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRValidateLegalParty(AbstractLogicQualityRule):
+    """
+    Check that legal party has not have data of natural party
+    """
+    _ERROR_01 = QRE_IGACR4008E01  # Business name must be filled in
+    _ERROR_02 = QRE_IGACR4008E02  # First last name must not be filled in
+    _ERROR_03 = QRE_IGACR4008E03  # First name must not be filled in
+    _ERROR_04 = QRE_IGACR4008E04  # Type of document must be NIT or Sequential
+    _ERROR_05 = QRE_IGACR4008E05  # Second last name must not be filled in
+    _ERROR_06 = QRE_IGACR4008E06  # Second name must not be filled in
+    _ERROR_07 = QRE_IGACR4008E07  # Genre field must not be filled in
+
```

Comment:
I think we should add another error for the marital status.

Reply:
That's right, change made in commit https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/ce34c9344a1bd20b2c0ddff94cbc1fa8935fe495